src/main/java/org/opentripplanner/routing/core/StateData.java

```diff
@@ -76,6 +76,10 @@
 
     protected ServiceDay serviceDay;
 
+    /**
+     * The mode of transit to use when not traveling on transit. This should be updated each time transitions happen to
```

Comment:
Maybe change to "The mode of travel to use when not traveling by public transit..."